### PR TITLE
Author Show

### LINKED ISSUE #28: User Story 14, Author Show Page
```
As a Visitor,
When I visit an author's show page
I see all books by that author
Each book should show:
- the book title
- the number of pages in the book
- the year the book was published
- a small image of the book cover

Each book should show a list of any other authors
(exclude this show page's author from that list)
```

DO THESE STORIES FIRST:
- User Story 12, Add a New Book to the System

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #15

child of #33

depends on #17

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in the functionality of an Author show page. Author show pages will contain all the pertinent information about that Author and their Books.
Each of their Books will be sectioned off, including their image, title, number of pages, year published and any other Co-authors.
Co-authors will only display if the book has multiple authors. Only the authors who are NOT the currently shown author will display.

Resolves #28

### CODE REVIEW

File: app/views/authors/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <img src="<%= book.image %>" alt="<%= book.title %>"/>
+    <h5><%= book.title %></h5>
+    <p>Pages: <%= book.pages %></p>
+    <p>Published: <%= book.year %></p>
+    <% if book.coauthors(@author).any? %>
+      <p>Co-authors: <%= book.coauthors(@author).join(', ') %></p>
+    <% end %>
+
+  </section>
+<% end %>
```
Comment (alexander-mathieu) [inline app/views/authors/show.html.erb:14]: Nice, glad we got rid of the map!
